src/ui/rideWindow.ts

```diff
@@ -115,6 +116,28 @@ export const rideWindow = window({
 				})
 			]
 		}),
+		groupbox({
+			text: "Ride name",
+			tooltip: "Edit the name of the ride",
+			content: [
+				label({
+					text: model._title,
+					height: 14
```

Comment:
14 is the default height of a label, so this line can be omitted.

Reply:
removed the line.